web/src/test/java/com/replica/RouterServerTest.java

```diff
@@ -350,11 +385,11 @@ private void checkTransitQuery(RouterOuterClass.PtRouteReply response,
         int observedStableEdgeIdCount = 0;
         double observedDistanceMeters = 0;
         for (RouterOuterClass.PtLeg streetLeg : streetLegs) {
-            assertTrue(streetLeg.getStableEdgeIdsCount() > 0);
+            assertTrue(streetLeg.getStableEdgeIdsCount() >= 0);
```

Comment:
One sliiightly suboptimal effect of the new empty legs is that a few of these checks get relaxed from `>` to `>=`. 

Given that `0` will now be a valid (and somewhat frequent) value for these properties, do we think a more explicit check that separates out the operator we use to check empty legs we generated ourselves vs. legs generated by GraphHopper is worth it?

I think we'd need to add a new bool flag to the `PtLeg` proto message type indicating whether or not a leg was an empty, Replica-generated leg or not. The only other way to currently check for empty legs is just checking if their stable edge ID list is empty, but this would miss a situation where GraphHopper returned a walk leg missing SEIDs (which shouldn't happen)

Reply:
I like that approach, even if imperfect it makes these checks a lot safer and should cover issues in core graphhopper unless something really funky occurs (graphhopper itself generates legs with zero duration, distance, and SEID count).

Made the update in [5447198](https://github.com/replicahq/graphhopper/pull/168/commits/54471982565332a5fe9cf50a0c386a4f9bf80282) to enforce `0` value checks for distance/duration in the case where there are no SEIDs returned, and to check for values `>0` otherwise